src/control_types.h

```diff
@@ -51,21 +51,22 @@ struct CursorControls_t
     // scroll up / down
     bool ScrollUp = false;
     bool ScrollDown = false;
+
     // call to easily send cursor offscreen (on disconnect, etc)
     inline void GoOffscreen()
     {
-        X = -128.;
-        Y = -128.;
+        X = -4000.0;
+        Y = -4000.0;
```

Comment:
Why not to make the sort of -9999999999999 (something ultra-huge and being constant in binary form to avoid possible false positives, or something, but right, Editor won't allow non-multiple 32 camera location being set).